tests/components/fronius/test_diagnostics.py

```diff
@@ -0,0 +1,31 @@
+"""Tests for the diagnostics data provided by the KNX integration."""
```

Comment:
Fronius integration

Reply:
Oh, thanks! I added the typo fix to another open PR for the Fronius integration as this needed a rebase anyway. https://github.com/home-assistant/core/pull/116132